js/templates/nav-data.hbs

```diff
@@ -42,7 +42,7 @@
       </div>
     </div>
     <div class="mega-heading">
-      <h3 class="mega-heading__title icon-heading--table"><a href="{{webAppUrl}}/advanced">Advanced data</a></h3>
+      <h3 class="mega-heading__title icon-heading--table"><a href="{{webAppUrl}}/advanced">Advanced data - all</a></h3>
     </div>
```

Comment:
Yep this is right. Not sure why it wasn't working. I'll review, but we should be good.